src/libpgmoneta/wf_cleanup.c

```diff
@@ -132,3 +133,12 @@ cleanup_teardown(int server, char* identifier, struct deque* nodes)
 
    return 0;
 }
+
+static int
+cleanup_execute_restore_incremental(int server, char* identifier, struct deque* nodes)
+{
+   char* directory_incremental = NULL;
+   directory_incremental = (char*)pgmoneta_deque_get(nodes, NODE_DIRECTORY);
+   pgmoneta_delete_directory(directory_incremental);
```

Comment:
Can this be done in the code above ? Then, we can remove this function

Reply:
Removed